modules.json

```diff
@@ -10,6 +10,7 @@
     "prebidServerBidAdapter",
     "consentManagement",
     "criteoBidAdapter",
+    "s2sTesting",
```

Comment:
what does this module do? why are we pushing something that looks related with testing to prod?